content/yaml/building-a-react-native-app.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Building a React Native app
+description: Building a React Native app with YAML.
+weight: 5
+---
+
+React Native is a cross-platform solution that allows you to build apps for both iOS and Android faster using a single language. When working with YAML, the basics are still the same, the build scripts are added to the `scripts` section in the [overall architecture](../yaml/yaml/#template).
+
+## Android
+
+Script for building an Android application:
+
+    - echo "sdk.dir=$HOME/programs/android-sdk-macosx" > "$FCI_BUILD_DIR/android/local.properties"
+    - cd android && ./gradlew bundleDebug
+
+## iOS
+
+Script for building an iOS application:
+
+    - xcode-project build-ipa --workspace "ios/MyReact.xcworkspace" --scheme "MyReact"
+
+## Testing
+
+To test an application, add the test script to the `scripts` section of your code. This is a basic example with jest, given that jest tests are defined in the `package.json` file:
+
+    npm test
```

Comment:
Duplicate

Reply:
fixed in 420381a6c3bdbcdef41e1b0cb4f45aaec50f173f